resource/WebGui/app/service/views/InMoov2Gui.html

```diff
@@ -942,25 +967,124 @@
             <button class="btn btn-default" type="button" ng-click="msg.stop()">
                 <span>
                     <img src="../stop.png"/>&nbsp;&nbsp;&nbsp;&nbsp;stop 
+                
+                
                 </span>
             </button>
             <button class="btn btn-default" ng-click="msg.disable()">disable</button>
-</button></td></tr>
+        </td>
+    </tr>
+
+<!-- begin -->
+            <!--
 <tr>
-    <td>virtualize&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<td>
+<b>enable system</b><br/>
+-->
+            <div id="table-wrapper">
+                <div id="table-scroll">
+                    <table class="table table-striped">
+                        <tbody>
+                            <tr>
+                                <td>virtualize</td>
+                                <td>
         <toggle width="30" height="28" ng-model="service.isVirtual" ng-change="!service.isVirtual?msg.setAllVirtual(false):msg.setAllVirtual(true)" on=" " off=" "></toggle>
     </td>
 </tr>
 <tr>
-    <td>simulator&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                                <td>head</td>
```

Comment:
I am not sure it's good to have the user enable a service before having it configured.
When head, arms, hands, torso are displayed in main panel, it might induce in starting the service when it has not been configured. That is the reason I have each skeleton buttons which leads the user to activate and then do the settings. (isActivated in our old ini, meant that when you would launch the scripts, only the isActivated parts would start. In Nixie this became a bit mixed up, because only when the part isActivated it can be configured...

Reply:
isActivated by convention is a status not a command
In Nixie you absolutely have the ability to configure before starting ... the difference is its done in yaml vs ini files ..
The challenge that isn't quite there is the configuration of the yaml done graphically